Makefile

```
@@ -133,11 +133,21 @@ curl:
 	curl -sS http://localhost:$(PORT)/api/v1/echo || true; echo
 
 postgres:
-	@echo "Creating postgres container..."; \
-	docker run -d --name postgres -p 5432:5432 -e POSTGRES_USER=admin -e POSTGRES_PASSWORD=dpm -e POSTGRES_DB=dpm postgres; \
-	echo "Done."
+	@echo "Creating postgres container..."
+	@docker run -d --name postgres -p 5432:5432 -e POSTGRES_USER=admin -e POSTGRES_PASSWORD=dpm -e POSTGRES_DB=dpm postgres
+	@echo "Done."
 
 postgres-stop:
-	@echo "Stopping postgres container..."; \
-	docker stop postgres; \
-	echo "Done."
+	@echo "Stopping postgres container..."
+	@docker stop postgres
+	@docker rm postgres
+	@echo "Done."
+
+ssh:
+	ssh root@${APP__SERVER__URL}
+
+poetry:
+	@pip install poetry
+
+auth-test:
```

```suggestion
auth-test: poetry
```
test 명령어에도 auth-test 의존성을 걸면 함께 테스트 진행할 수 있을 것 같습니다

---

의존성 추가했습니다!

make test의 경우 e2e 테스트를 목적으로 만들어진 것이라, 따로 분리해두는 게 나을 것 같습니다!
굳이 나누면 auth-test랑, e2e-test 이런식으로 쪼갤 수 있겠네요 ㅎㅎ